apps/root/src/constants/aggregator.ts

```diff
@@ -120,6 +120,9 @@ export const DEFAULT_AGGREGATOR_SETTINGS: {
   sourceTimeout: TimeoutKey.balance,
 };
 
+const tempSdk = buildSDK();
+const aggSupportedChains = tempSdk.quoteService.supportedChains();
```

Comment:
If this is aggSupportedChains, can we export this directly? We would erase the manual task of adding chains to the `AGGREGATOR_SUPPORTED_CHAINS: ChainId[]`